schema.graphql

```diff
@@ -47,19 +47,40 @@ type Query {
   mostFollowedChannelsAllTime(limit: Int!): [ChannelFollowsInfo!]
 
   """Get most viewed list of categories"""
-  mostViewedCategories(limit: Int, period: Int!): [EntityViewsInfo!]
+  mostViewedCategories(
+    limit: Int
+
+    """
+    Period is specified in days and must take one of the following values: 7, 30
+    """
+    period: Int!
+  ): [EntityViewsInfo!]
 
   """Get most viewed list of categories of all time"""
```

Comment:
Let's update the description in the code to `Get list of most viewed categories of all time`